region/brinstar/blue.json

```diff
@@ -1338,6 +1539,146 @@
                   "requires": [ "Morph" ]
                 }
               ]
+            },
+            {
+              "id": 5,
+              "strats": [
+                {
+                  "name": "G-Mode Morph",
+                  "notable": false,
+                  "requires": [
+                    "f_ZebesAwake",
+                    {"comeInWithGMode": {
+                      "fromNodes": [1],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    {"or": [
+                      "SpringBall",
+                      "Bombs",
+                      {"ammo": {"type": "PowerBomb","count": 1}},
+                      {"enemyDamage": {
+                        "enemy": "Geemer (blue)",
+                        "hits": 1,
+                        "type": "contact"
+                      }},
+                      {"comeInWithGMode": {
+                        "fromNodes": [1],
+                        "mode": "any",
+                        "artificialMorph": true,
+                        "mobility": "immobile"
```

Comment:
Not that its technically wrong, but immobile would require direct and should lead to the direct junction.